src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -650,8 +971,68 @@
 												<SolidColorBrush x:Key="TextControlBackgroundFocused" Color="{ThemeResource SolidBackgroundFillColorSecondary}" />
 											</TextBox.Resources>
 										</TextBox>
+
 									</Grid>
 
+									<!--  Item Git Status  -->
+									<TextBlock
+										x:Name="ItemGitStatusTextBlock"
```

Comment:
It would be more helpful to have a tooltip showing what the status letters mean.
![image](https://github.com/files-community/Files/assets/66369541/87ea80a1-039e-47b2-b7ed-9da217829dac)